src/SidePanel/OtherSettingsView.jsx

```diff
@@ -50,30 +48,29 @@ import {
 } from '../reducers/settingsReducer';
 import { getIsRunning } from '../reducers/testReducer';
 import { fromPCA } from '../utils/boards';
+import Dropdown from './dropdown/Dropdown';
+import DropdownItem from './dropdown/DropdownItem';
 
 const phyTypeView = (boardType, phy, onPhyUpdated, isRunning) => {
     const compatibility = fromPCA(boardType);
     const items = Object.keys(compatibility.phy).map(keyname => (
-        <Dropdown.Item
+        <DropdownItem
             key={keyname}
+            title={DTM_PHY_STRING[compatibility.phy[keyname]]}
             eventKey={keyname}
-            onSelect={evt => onPhyUpdated(compatibility.phy[evt])}
-        >
-            {DTM_PHY_STRING[compatibility.phy[keyname]]}
-        </Dropdown.Item>
+            onSelect={eventKey => onPhyUpdated(compatibility.phy[eventKey])}
```

Comment:
I might be mistaken, but I think it would work fine to just do 
`onSelect={() => onPhyUpdated(compatibility.phy[keyname])` here, which would mean you can avoid passing in the `eventKey` prop to the `DropdownItem` component entirely. Not a huge thing, but something to consider.

Reply:
I will test it out